mypyc/irbuild/ll_builder.py

```diff
@@ -927,23 +927,8 @@ def new_list_op_with_length(self, length: Value, line: int) -> Value:
         return self.call_c(new_list_op, [length], line)
 
     def new_list_op(self, values: List[Value], line: int) -> Value:
-        length = Integer(len(values), c_pyssize_t_rprimitive, line)
```

Comment:
Is `make_list` a good name? since we have another `new_list_op_with_length` for creating empty list.

Reply:
Ummm, I saw we have different namings now like `make_dict` and `new_set_op`, I think we should adapt one and stick to it, like `make_xxx` or `new_xxx_op` but not both.